src/test/resources/graph/real/get-ccloud-environment-flink-query.graphql

```diff
@@ -0,0 +1,13 @@
+query {
+  ccloudConnectionById(id: "ccloud-dev") {
+    environments {
+      flinkComputePools {
+        id
+        display_name
+        provider
+        region
+        max_cfu
+      }
+    }
+  }
+}
```

Comment:
Should this query target a different environment than the one in `src/test/resources/graph/real/get-ccloud-environment-empty-flink-query.graphql`?